packages/graph/hash_graph/lib/graph/src/store/postgres/ontology/read.rs

```diff
@@ -82,7 +82,27 @@ where
     type Query<'q> = Filter<'q, T::Inner>;
 
     async fn read<'f: 'q, 'q>(&self, filter: &'f Self::Query<'q>) -> Result<Vec<T>, QueryError> {
-        let mut compiler = SelectCompiler::with_default_selection();
+        let versioned_uri_path =
+            <<T::Inner as QueryRecord>::Path<'q> as OntologyPath>::versioned_uri();
+        let schema_path = <<T::Inner as QueryRecord>::Path<'q> as OntologyPath>::schema();
+        let owned_by_id_path = <<T::Inner as QueryRecord>::Path<'q> as OntologyPath>::owned_by_id();
+        let created_by_id_path =
+            <<T::Inner as QueryRecord>::Path<'q> as OntologyPath>::created_by_id();
+        let updated_by_id_path =
+            <<T::Inner as QueryRecord>::Path<'q> as OntologyPath>::updated_by_id();
```

Comment:
The path selection is.. complicated, but also not worth investing time into fixing right now if this is in-flux :)